### ISSUE TITLE
Correction based on flow rate instead of length alone

### ISSUE THREAD

**Issue by dewi-ny-je** [Alexander-T-Moss/Small-Area-Flow-Comp#14]
I wrote a feature request on Orca Slicer: https://github.com/SoftFever/OrcaSlicer/issues/7390

Now that adaptive PA can already improve short extrusions A LOT, the remaining minor variations in flow depend on the non linear flow extrusion due to the extruders.
See video posted there.

Would you be interested in modifying the script to compensate based on exclusively the max flow achieved along each extrusion length, using a correction graph (input data) like the ones measured in the video I posted taking into account line width layer height, acceleration used.
So basically it would be flat up to around 9 mm3/s then more or less linearly decrease until it collapses (and no compensation is possible).

Coding is not my thing so if you were able to modify the script I would perform all testing of the results.

**Comment by mrdROb** [comment]
Do we have any progress? I am looking for ANY solution to the problem you explained... Whether Klipper modification, or Slicer, or even G-Code post script...
